<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Modules">
    <property name="title">
     <string>&amp;Modules</string>
    </property>
    <addaction name="actionProgram"/>
    <addaction name="actionDSP_Engine"/>
   </widget>
   <widget class="QMenu" name="menu_Debug">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <addaction name="action_Enable"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Modules"/>
   <addaction name="menu_Debug"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionProgram">
   <property name="text">
    <string>&amp;Program...</string>
   </property>
  </action>
  <action name="actionDSP_Engine">
   <property name="text">
    <string>DSP &amp;Binary...</string>
   </property>
  </action>
  <action name="action_Enable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Enable</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
